docs/modules/a2c.rst

```diff
@@ -76,6 +76,49 @@ Train a A2C agent on ``CartPole-v1`` using 4 environments.
       env.render()
 
 
+Scaling with CPU Cores
+----------------------
+
+The A2C algorithm allows for scaling with multiple CPU cores through parallel processing.
+To enable this feature, you'll have to override the vector environment implementation
+instantiated by ``make_vec_env()`` such that it creates a ``SubprocVecEnv``.
+
+The default ``DummyVecEnv`` implementation will only run a single thread for processing actions
+and collecting observations / rewards. With ``SubprocVecEnv``, each environment is running in its
+own thread, facilitating parallel processing for superior CPU utilization.
```

Comment:
I would replace this by ``"(See :ref:`Vectorized Environments <vec_env>`)"``

Reply:
I agree, putting a link for further information on VecEnvs is a good idea. 

But I wouldn't leave out the short summary on how the different VecEnvs work. I think it fits there quite well.

But yeah, if you don't like it due to duplication for ease of maintainability, go ahead and delete it. I don't mind.

Reply:
you could also link to the Notebook about multiprocessing

Reply:
@araffin Feel free to make your suggestion. I'm done for my part. You can do whatever you want with my changes 😄